src/FplBot.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:Boolean x:Key="/Default/UserDictionary/Words/=Gameweek/@EntryIndexedValue">True</s:Boolean></wpf:ResourceDictionary>
```

Comment:
What dis?  Resharper dictionary?

Reply:
Yep, Resharper gave me squigglies on "Gameweek". Could've just renamed to "GameWeek" everywhere, but typing it like we do looks better imo.